go/vt/vtgate/planbuilder/operators/aggregation_pushing.go

```diff
@@ -369,14 +369,14 @@ func pushAggregationThroughApplyJoin(ctx *plancontext.PlanningContext, rootAggr
 
 	columns := &applyJoinColumns{}
 	output, err := splitAggrColumnsToLeftAndRight(ctx, rootAggr, join, !join.JoinType.IsInner(), columns, lhs, rhs)
-	join.JoinColumns = columns
```

Comment:
This bug was not related to the rest of the PR, but it caused tests to fail, so it had to go.